src/module.c

```diff
@@ -10980,8 +10980,9 @@ int moduleUnregisterFilters(ValkeyModule *module) {
  * If multiple filters are registered (by the same or different modules), they
  * are executed in the order of registration.
  */
-ValkeyModuleCommandFilter *
-VM_RegisterCommandFilter(ValkeyModuleCtx *ctx, ValkeyModuleCommandFilterFunc callback, int flags) {
+ValkeyModuleCommandFilter *VM_RegisterCommandFilter(ValkeyModuleCtx *ctx, 
+                                                    ValkeyModuleCommandFilterFunc callback, 
```

Comment:
```suggestion
ValkeyModuleCommandFilter *VM_RegisterCommandFilter(ValkeyModuleCtx *ctx,
                                                    ValkeyModuleCommandFilterFunc callback,
```
Sigh, missed some whitespace.